catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/predicate/TemporalPredicate.java

```diff
@@ -102,22 +100,31 @@ public boolean matches(Event properties) {
         if (entry != null) {
             LOGGER.debug("entry id: {}", entry.getId());
 
-            switch (this.type) {
-            case modified:
+            switch (this.type.getAttributeName()) {
```

Comment:
❗️ You should be able to keep using the enum like before and simply add `case metacard_modified` and `case metacard_created` to the `switch` statement.